src/mqt/bench/benchmark_generation.py

```
@@ -75,20 +75,26 @@ def _get_circuit(
     return qc
 
 
-def _create_mirror_circuit(qc_original: QuantumCircuit, inplace: bool = False) -> QuantumCircuit:
+def _create_mirror_circuit(
+    qc_original: QuantumCircuit, inplace: bool = False, target: Target | None = None, optimization_level: int = 2
```

Ruff (0.14.0)

Boolean-typed positional argument in function definition

(FBT001)